src/assignment/views/AssignmentEdit.tsx

```diff
@@ -634,6 +652,28 @@ const AssignmentEdit: FunctionComponent<DefaultSecureRouteProps<{ id: string }>>
 					</StickyEdgeBar>
 				)}
 			</Container>
+
+			{original && (
+				<ConfirmSlice
+					isOpen={!!isConfirmSliceOpen}
+					assignment={original}
+					block={getConfirmSliceBlock as AssignmentBlock}
```

Comment:
is `getConfirmSliceBlock` a function or an object?
shouldn't this be renamed to: confirmSliceBlock?

Reply:
De `get`-prefix zorgt hier voor eenduidigheid met de rest van de state-variabelen & functies in
```
const [
  isConfirmSliceOpen,
  setConfirmSliceOpen,
  getConfirmSliceBlock,
  setConfirmSliceBlock,
] = useSingleEntityModal<Pick<AssignmentBlock, 'id'>>();
```

imo is het onderstaande veel slechter scan-able.
```
const [
  isConfirmSliceOpen,
  setConfirmSliceOpen,
  confirmSliceBlock,
  setConfirmSliceBlock,
] = useSingleEntityModal<Pick<AssignmentBlock, 'id'>>();
```